lib/synapse_pay_rest/http_client.rb

```diff
@@ -53,6 +53,10 @@ def update_headers(user_id: nil, oauth_key: nil, fingerprint: nil, client_id: ni
       end
     end
 
+    def add_idempotency_key(idempotency_key:)
```

Comment:
I have been trying to add test coverage with each PR.  So when I touch this code base, I sometimes throw in an extra test.